lms/envs/test.py

```diff
@@ -582,3 +582,7 @@
 COURSE_CATALOG_API_URL = 'https://catalog.example.com/api/v1'
 
 COMPREHENSIVE_THEME_DIRS = [REPO_ROOT / "themes", REPO_ROOT / "common/test"]
+
+ENV_TOKENS = {
+    'LMS_ROOT_URL': 'https://localhost:8000',
+}
```

Comment:
For LMS you commit only set it in `lms/envs/bok_choy.env.json`. We would need same config for LMS as in CMS.